pkgs/by-name/na/nats-surveyor/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub, }:
+buildGoModule rec {
+  pname = "nats-surveyor";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "nats-io";
+    repo = "nats-surveyor";
+    rev = "v${version}";
+    hash = "sha256-7HJgSwsB7m1Y6U2+ZMQ0JHHdLge5v7wLjH5chlhB0F0=";
+  };
+
+  vendorHash = "sha256-NjwUiommvCy5arr1QCcDjqnxuLwN3fsj3C1O6hZL2jw=";
+
+  meta = with lib; {
+    homepage = "https://github.com/nats-io/nats-surveyor";
+    description = "NATS Monitoring, Simplified.";
+    longDescription = ''
+      NATS surveyor polls the NATS server for Statz messages to
+      generate data for Prometheus. This allows a single exporter
+      to connect to any NATS server and get an entire picture of
+      a NATS deployment without requiring extra monitoring
+      components or sidecars.
+    '';
+    license = licenses.asl20;
+    maintainers = with maintainers; [ lblasc ];
+  };
```

```suggestion
    mainProgram = "nats-surveyor";
  };
```

> `meta.mainProgram` must be set to the name of the executable which facilitates the primary function or purpose of the package, if there is such an executable in $bin/bin/ (or $out/bin/, if there is no "bin" output).

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

It should be set `meta.mainProgram`.